bootstrap/dev/venv.sh

```diff
@@ -11,3 +11,5 @@ export VENV_ARGS="--python python2"
   -e letsencrypt-nginx \
   -e letshelp-letsencrypt \
   -e letsencrypt-compatibility-test
+
+find . -iname '*.pyc' -exec rm '{}' ';'
```

Comment:
I think `-e` in the shellbang is not really safe enough for this. For an example, if the user types in `bash /path/to/bootstrap/dev/venv.sh`